.zuul.yaml

```diff
@@ -1,6 +1,23 @@
 ---
-- project:
-    name: infrawatch/feature-verification-tests
+- job:
+    name: feature-verification-tests-osp18
+    parent: telemetry-operator-multinode-autoscaling
+    description: |
```

Comment:
❌ [feature-verification-tests-osp18 ](https://review.rdoproject.org/zuul/build/1fd420ae5ac243a9b310924f6ea6b118)ERROR Project github.com/openstack-k8s-operators/dataplane-operator does not have the default branch master in 5s

You will need to add the dataplane operator to the job definition with the override checkout

Either you can add it here, or in the telemetry-job-multinode-autoscaling definition in telemetry-operator.

required-projects:
    - name: openstack-k8s-operators/dataplane-operator
       override-checkout: main 
       
Example is here: https://github.com/infrawatch/service-telemetry-operator/blob/853e4574b5d3c93faa2b9001f8bf18f0a9eddc9e/.zuul.yaml#L31C1-L39C32